pkg/ottl/ottlfuncs/README.md

```diff
@@ -136,19 +135,18 @@ Examples:
 
 - `merge_maps(attributes, ParseJSON(body), "upsert")`
 
-
 - `merge_maps(attributes, ParseJSON(attributes["kubernetes"]), "update")`
 
-
 - `merge_maps(attributes, resource.attributes, "insert")`
 
 ### replace_all_matches
 
-`replace_all_matches(target, pattern, replacement)`
+`replace_all_matches(target, pattern, replacement, hashFunction)`
```

Comment:
Lets word this more generically.  Although hashing is a primary goal, the function can take in any Converter that accepts a string and returns a string. We can callout some common uses cases like hashing.

Reply:
Made this change to the readme.